zebra-consensus/src/block/check.rs

```diff
@@ -25,3 +30,40 @@ pub fn is_coinbase_first(block: &Block) -> Result<(), Error> {
     }
     Ok(())
 }
+
+/// [3.9]: https://zips.z.cash/protocol/protocol.pdf#subsidyconcepts
+pub fn is_subsidy_correct(block: &Block) -> Result<(), Error> {
```

Comment:
I think the naming of these functions is good. Having an incorrect subsidy is an error in block verification.  I don't think we should add a prefix, because the naming already fits with the module layout:
```rust
            check::is_coinbase_first(&block)?;
            check::is_subsidy_correct(&block)?;
```

Reply:
good point on the `check` module. what about if we just switch it around so the `is_` isn't leading, to more clearly differentiate it from `is_` functions that return bool?

```rust
            check::coinbase_is_first(&block)?;
            check::subsidy_is_correct(&block)?;
```

Reply:
renamed all `check::*` functions at https://github.com/ZcashFoundation/zebra/pull/1051/commits/32dd22d537608e594e4011a77d4899fd5ab5dff8

Reply:
Changing the names of these functions is ok, but it isn't really related to the block subsidy. It also caused a merge conflict with the #1146 difficulty PR.

Let's try to submit separate PRs for unrelated cleanups?

(And let's try to submit smaller PRs overall: they're easier and faster to review and merge. This will become more important for the first alpha release, because we'll all be focused on the sync, state, and difficulty code.)